lib/server.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const routes = require('./routes/people-routes.js');
+
+let db = require('./db.js');
+app.use(express.json());
+
+// Default Route
+app.get('/', (req, res, next) => {
+  res.send('Homepage');
+});
+
+//functions
+// have a function that sets req.requestTime
+const requestTime = (req, res, next) => {
+  req.requestTime = (new Date().toUTCString());
+  console.log(req.requestTime);
+  next();
+};
+
+const eachRequest = (req, res, next) => {
+  console.log(`req.path, req.method,requestTime and req.message:
+  ${req.path}, ${req.method}, ${req.requestTime}, ${req.setMessage}`);
+  next();
+};
+//MiddleWare
+app.use(eachRequest);
+app.use(requestTime);
+// Routes
+app.use(routes);
```

Comment:
because you've been using `app.` in your modular route files, you're getting some weird behavior. You should have used `router.` in your modular route files, and here you would do `app.use('/people', routes); ` and then another line that says `app.use('/teams', routes);`